unittests/tools/test_threat_composer_parser.py

```diff
@@ -0,0 +1,81 @@
+from django.test import TestCase
+
+from dojo.models import Test
+from dojo.tools.threat_composer.parser import ThreatComposerParser
+
+
+class TestThreatComposerParser(TestCase):
+
+    def test_threat_composer_parser_with_no_threat_has_no_findings(self):
+        testfile = open("unittests/scans/threat_composer/threat_composer_zero_threats.json")
+        parser = ThreatComposerParser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        self.assertEqual(0, len(findings))
+
+    def test_threat_composer_parser_with_one_threat_has_one_finding(self):
+        testfile = open("unittests/scans/threat_composer/threat_composer_one_threat.json")
+        parser = ThreatComposerParser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+
+        self.assertEqual(1, len(findings))
+
+        with self.subTest(i=0):
+            finding = findings[0]
+            self.assertEqual("lorem ipsum", finding.title)
+            self.assertEqual("High", finding.severity)
+            self.assertIsNotNone(finding.description)
+            self.assertTrue(
+                "Assumption" in str(finding.description),
+            )
```

Comment:
```suggestion
            self.assertIn("Assumption", str(finding.description))
```
It is recommended to use `assertTrue`/`assertFalse` only if there is no better alternative. I believe that `assertIn` is quite good alternative 😄
It is also handy for debugging (if a substring is not in a string, whole string will be printed in an error message)

Reply:
Changed, thx